util/DHI.Mike1D.MikeIO/LTSResultEvents.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.Collections.Generic;
+
+namespace DHI.Mike1D.MikeIO
+{
+  /// <summary>
+  /// List of LTS events stored in result files.
+  /// </summary>
+  public class LTSResultEvents : List<LTSResultEvent>
+  {
+    /// <summary>
+    /// Sort the event list on the first value of each event
+    /// </summary>
+    public void SortOnValue()
+    {
+      Sort(CompareValue);
+    }
+
+    /// <summary>
+    /// Sort on first value, and if the same, then on time.
+    /// </summary>
+    public int CompareValue(LTSResultEvent e1, LTSResultEvent e2)
```

Comment:
Why not use [`IComparable<LTSResultEvent>`](https://learn.microsoft.com/en-us/dotnet/api/system.icomparable-1?view=net-8.0)?

Reply:
In Mike1D there is already LTSEvents class in DHI.Mike1D.Engine, which has this type of structure. Now I don't want to change it there. Additionally, there are two ways of sorting which need to be performed. SortOnValue and SortOnTimePeriod. Of course one could say, that it is possible to have LTSResult**s**Extreme and LTSResult**s**Periodic, which implement IComparable<LTSResultEvent> and then has the Sort method. This holds only if LTSResultExtreme does not need sorting on time in DHI.Mike1D.Engine and I think that is the case.